src/AppInstallerSharedLib/Public/winget/Yaml.h

```diff
@@ -104,6 +104,10 @@ namespace AppInstaller::YAML
             return m_sequence->emplace_back(std::forward<Args>(args)...);
         }
 
+        // Merges sequence nodes. If both sequence have the specified key with the same value
+        // they will get merged together. All elements in sequence must have the key.
+        void MergeSequenceNode(Node& other, std::string_view key);

```

Comment:
Same as other places; I think we probably want case-insensitivity.  It should probably be a parameter at this level.